src/lib/journey/callbacks/kba/kba-create.stories.js

```diff
@@ -38,71 +38,50 @@ Interaction.play = async ({ canvasElement }) => {
 
   const cb = step.getCallbackOfType(CallbackType.KbaCreateCallback);
 
-  await new Promise((resolve) => setTimeout(resolve, 1000));
-
   await userEvent.tab();
 
-  await new Promise((resolve) => setTimeout(resolve, 1000));
-
   const selectEl = canvas.getByLabelText('Select a security question', {
     selector: 'select',
   });
-  await userEvent.selectOptions(selectEl, '0', { delay: 200 });
+  await userEvent.selectOptions(selectEl, '0', { delay: 100 });
 
   await expect(canvas.queryByLabelText('Custom security question')).toBeNull();
 
-  await new Promise((resolve) => setTimeout(resolve, 1000));
-
   await userEvent.tab();
 
-  await new Promise((resolve) => setTimeout(resolve, 1000));
-
   const answerEl = canvas.getByLabelText('Security answer', {
     selector: 'input',
   });
 
   await userEvent.type(answerEl, 'Red', {
-    delay: 200,
-  });
-
-  await new Promise((resolve) => setTimeout(resolve, 1000));
-
-  await userEvent.selectOptions(selectEl, '2', { delay: 200 });
+    delay: 100,
+  }),
+    await waitFor(() => userEvent.selectOptions(selectEl, '2', { delay: 100 }));
```

Comment:
I was originally using the `delay` and the `setTimeout` to mimic a user actually typing and clicking to actually see the interaction in human time. If we make them really fast and remove the `setTimeout`s in between actions, I doubt anyone's going to be able to observe what's really happening. 

If we do that, let's just remove them all together (both `delay`s and `setTimeout`s). They are not really needed, and I don't think anyone's using them to validate UI behavior in Storybook itself.

Plus, aren't these equivalent?

```
await waitFor(() => userEvent.selectOptions(selectEl, '2', { delay: 100 }));
```
```
await userEvent.selectOptions(selectEl, '2', { delay: 100 });
```

Anyway, I'd just drop it all to this:

```
await userEvent.selectOptions(selectEl, '2');
```

Reply:
They aren't equivalent. I used waitFor because of failures. but i can look more into this if we want.

waitFor will retry until the expectation is satisfied. I only used waitFor because this test was failing after upgrading and it was how i got it to work. i saw it used in storybook docs and opted to try it.

Reply:
I wonder if `selectEl` isn't rendered on the page at that moment. Maybe the unintentional side-effect of the delays gave it time to render, but upon removal of the delays, the element doesn't not exist? I say that because we use the `await userEvent.selectOptions()` in other places without the need for `waitFor`, so something must be off here.